src/sbvr-api/odata-response.ts

```diff
@@ -158,9 +162,17 @@ export const process = async (
 	);
 
 	const odataIdField = sqlNameToODataName(table.idField);
+	const webResourceHandler = getWebresourceHandler();
 	for (const row of rows) {
 		for (const fieldName of processedFields) {
 			row[fieldName] = fetchProcessingFields[fieldName](row[fieldName]);
+			if (
+				webResourceHandler &&
+				isWebResourceField(fieldName, table) &&
```

Comment:
I don't know about the performance, but the table is static for each iteration, potentially the isWebResourceFiled can be wrapped into a table specific function and in the loop the tableSpecific function is executed?

Reply:
isWebResourceFieled could be optimised into a 
`entityContainsWebResourceField` to select a table being a webResource containing table, instead of iterating over all fields on the table for the specific fieldname?